.github/workflows/main.yaml

```diff
@@ -179,6 +179,56 @@ jobs:
             /tmp/cloudcore.log
             /tmp/edgecore.log
 
+  quic_e2e_test:
```

Comment:
I think we can reuse job `e2e_test` to run `quic-e2e-test` through the github action feature:
````    
strategy:
      matrix:
````
One example can be retrieved from release.yaml.